config/parser.go

```diff
@@ -313,6 +314,7 @@ func (parser ConfigParser) getDefaultConfig() Config {
 			},
 		},
 		ConfirmQuit: false,
+		ShowAuthorRole: false,
```

Comment:
I'm not seeing where this is used? perhaps you intended to use it in `GetAuthor()`?